src/styles/vendor/el-plus/_button.scss

```
@@ -0,0 +1,20 @@
+@use '@/styles/resources/variables' as *;
+
+.el-button {
+  height: auto;
+  color: $color--text;
+  padding: 15px 10px;
+
+
+
+  &.el-button--primary {
+    border: none !important;
+    background: $color--primary;
+
+    &.background{
```

```suggestion
    &.background {
```

Везде отступы поставь :(